shell/platform/darwin/ios/framework/Source/FlutterAppDelegate.mm

```diff
@@ -40,21 +40,21 @@ - (BOOL)application:(UIApplication*)application
 
 // Returns the key window's rootViewController, if it's a FlutterViewController.
 // Otherwise, returns nil.
-- (FlutterViewController*)rootFlutterViewController {
++ (FlutterViewController*)rootFlutterViewController {
   UIViewController* viewController = [UIApplication sharedApplication].keyWindow.rootViewController;
   if ([viewController isKindOfClass:[FlutterViewController class]]) {
     return (FlutterViewController*)viewController;
   }
   return nil;
 }
 
++ (void)handleStatusBarTouches:(NSSet*)touches withEvent:(UIEvent*)event {
+  [self.rootFlutterViewController handleStatusBarTouches:event];
+}
+
 - (void)touchesBegan:(NSSet*)touches withEvent:(UIEvent*)event {
```

Comment:
Let's also add UIResponder's generics to the NSSet here and handleStatusBarTouches: : `NSSet<UITouch *> *`